xbmc/settings/AdvancedSettings.cpp

```diff
@@ -298,8 +298,8 @@ void CAdvancedSettings::Initialize()
   m_iVideoLibraryRecentlyAddedItems = 25;
   m_bVideoLibraryCleanOnUpdate = false;
   m_bVideoLibraryUseFastHash = true;
-  m_bVideoLibraryImportWatchedState = false;
-  m_bVideoLibraryImportResumePoint = false;
+  m_bVideoLibraryImportWatchedState{true};
```

Comment:
But my gcc 10 does not like that at all

Reply:
shouldnt be a problem, whats the error?

Reply:
ofc...because this is wrong (thought this was the header file).
Sorry @DaVukovic. Please keep it as you had it before

```suggestion
  m_bVideoLibraryImportWatchedState = true;
  m_bVideoLibraryImportResumePoint = true;
```

You could also simply drop this two lines and initialize them to true in the header file (in AvancedSettings.h):

```
bool m_bVideoLibraryImportWatchedState{true};
bool m_bVideoLibraryImportResumePoint{true};
```

Reply:
```
/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/xbmc/settings/AdvancedSettings.cpp: In member function 'void CAdvancedSettings::Initialize()':
/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/xbmc/settings/AdvancedSettings.cpp:310:36: error: expected ';' before '{' token
  310 |   m_bVideoLibraryImportWatchedState{true};
      |                                    ^
      |                                    ;
/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/xbmc/settings/AdvancedSettings.cpp:310:3: warning: statement has no effect [-Wunused-value]
  310 |   m_bVideoLibraryImportWatchedState{true};
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/xbmc/settings/AdvancedSettings.cpp:311:35: error: expected ';' before '{' token
  311 |   m_bVideoLibraryImportResumePoint{true};
      |                                   ^
      |                                   ;
/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/xbmc/settings/AdvancedSettings.cpp:311:3: warning: statement has no effect [-Wunused-value]
  311 |   m_bVideoLibraryImportResumePoint{true};
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [build/settings/CMakeFiles/settings.dir/build.make:82: build/settings/CMakeFiles/settings.dir/AdvancedSettings.cpp.o] Error 1
make[2]: Leaving directory '/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/build'
make[1]: *** [CMakeFiles/Makefile2:7954: build/settings/CMakeFiles/settings.dir/all] Error 2
make[1]: Leaving directory '/home/xbian/xbian-package-xbmc/build/rpi4-nexus/working/build'
make: *** [Makefile:149: all] Error 2
```

Reply:
it IS a problem!

Reply:
whoops

Reply:
you should have a look at your outputs of the check build

Reply:
> you should have a look at your outputs of the check build
> 
> 

It was an honest mistake. It was also caught by our CI.